cache/models/samples.js

```diff
@@ -647,6 +649,7 @@ module.exports = {
       }
 
       cleanQueryBodyObj(reqBody);
+      aspectObj = aspObj;
```

Comment:
moved the       aspectObj = sampleStore.arrayStringsToJson(
        aspObj, constants.fieldsToStringify.aspect
      );

out of the value check